### PR TITLE
fix(cirriculum): tell campers to use leading space in rpg step 120

### LINKED ISSUE #53198: RPG - Step 120: Text inside code block is visually missing the starting space
### Describe the Issue

The text `You attack it with your <weapon>.` has a space in front of it that you can't see in the challenge text.

Not sure if it has to be a fenced code block (three backticks) for the space to show up. I assume it is because it is missing the `pre` element when it is just a single backtick code block. Maybe there is a setting as well that we can enable for it?

> On a new line, add the string
> 
> ``` 
>  You attack it with your <weapon>
> ```
> 
> to the text value, replacing `<weapon>` with the player's current weapon.

That or we need to mention it in the text or hints.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-120

### Your code

```



```

### Expected behavior

To see the spaces in the required text.

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

Forum: https://forum.freecodecamp.org/t/learn-basic-javascript-by-building-a-role-playing-game-step-120

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #53198

In the code base, this is actually step 120. All I did was add a line of text clarifying to the camper they should indeed add a leading space to the new line of text. Hopefully this is enough. If not, I'll be happy to see what else I could do.

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ dashedName: step-119
 
 # --description--
 
-On a new line, add the string ` You attack it with your <weapon>.` to the `text` value, replacing `<weapon>` with the player's current weapon.
+On a new line, add the string ` You attack it with your <weapon>.` to the `text` value, replacing `<weapon>` with the player's current weapon. Additionally, remember that this line of text starts
+with a space so it will properly display.
```
Comment (jdwilkin4) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md:11]: I don't think this needs to be a on new line here

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Let's add in the code block to so the camper can just copy the text like mentioned in the issue here 

https://github.com/freeCodeCamp/freeCodeCamp/issues/53198#issuecomment-1899028974

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md (hunk 1 above)
Comment (a2937) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md]: Thank you pointing that out. I couldn't actually tell an entire new line was added unnecessarily in my code editor.

**Review by JoyShaheb — APPROVED** [review]
looks good to me !

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**